net.kubux.Muehle.yml

```diff
@@ -0,0 +1,23 @@
+app-id: net.kubux.Muehle
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: muehle
+modules:
+  - name: muehle
+    buildsystem: simple
+    build-commands:
+      - tar -xjpvf muehle.tbz -C /
+    sources:
+      - type: file
+        url: http://www.kubux.net/muehle/muehle.tbz
```

Comment:
this as a file without a version. Which might not be stable when it gets updated for a new version. (changing the sha256)